HW5/TarasYarmus/12.py

```diff
@@ -0,0 +1,48 @@
+def rectangle(arg1, arg2):
+    '''
+    This function calculate square of the rectangle
+    Type: float
+    Input: your arguments
+    '''
+    return float(arg1*arg2)
+
+
+def triangle(height, basis):
+    '''
+    This function calculate square of the triangle
+    Type: float
+    Input: your arguments'''
+    return float(height/2)*basis
+
+
+def circle(r):
+    '''
+    This function calculate square of the circle
+    Type: float
+    Input: enter number for first argument "r"'''
+    return float(3.14*r)**2
+
+question = input(
+    '''
+    Hello! I will calculate square by your arguments! 
+    Calculate square for rectangle: press 1.
+    Calculate square for triangle: press 2.
+    Calculate square for circle: press 3.
+    Make your choice: ''')
+
+if question == '1':
+    rectangle1 = float(input('Enter first side length: '))
```

Comment:
it is better to move these lines to the appropriate functions

Reply:
Moved to the appropriate function.